firmware/rhizome.ino

```diff
@@ -104,13 +105,13 @@ void loop() {
 //        ((Ohmbrewer::Thermostat*)sprouts.at(0))->getElement()
 //                                               ->setState(!((Ohmbrewer::Thermostat*)sprouts.at(0))->getElement()
 //                                                                                                  ->getState()); // The Thermostat's HeatingElement
-        ((Ohmbrewer::RIMS*)sprouts.front())->getTube()
+        ((Ohmbrewer::Still*)sprouts.front())->getTherm()
                                            ->getElement()
                                            ->toggleState();
 //        navi->publish(); // (*)
         lastUpdate = millis();
     }
-    ((Ohmbrewer::RIMS*)sprouts.front())->getTube()->getSensor()->doWork(); //Temp patch to make sensor read in TUBE
+    ((Ohmbrewer::Still*)sprouts.front())->getColumnSensor()->doWork(); //Temp patch to make sensor read in TUBE
```

Comment:
This should call work() instead of doWork(). Not a showstopper, but just sayin'.